commit eb7cdc667273b7eb8a918cc6e35a4106f9fa5b1b (refs/changes/35/3435/1)
Author: Robert Mustacchi <rm@joyent.com>
Date:   2018-02-21T17:56:13+00:00 (1 year, 8 months ago)
    
    backout: RICHMOND-141 Allocation table headers are still busted (incorrectly merged)
    
    Change-Id: I4dd9e7a1b66dcd248f2ce0d36a34c89f2e720b56

diff --git a/allocations.md b/allocations.md
index 8cf2517..12fecd9 100644
--- a/allocations.md
+++ b/allocations.md
@@ -5,7 +5,7 @@
 -->
 
 <!--
-    Copyright (c) 2018, Joyent, Inc.
+    Copyright (c) 2017, Joyent, Inc.
 -->
 
 # Allocations
@@ -15,8 +15,8 @@ which are used as part of SMBIOS and other product naming.
 
 ## Products, SKU, Common Name
 
-| Common Name | Product | SKU |
-|-------------|---------|-----|
+| SKU | Common Name | Product ID |
+|-----|-------------|------------|
 | Richmond A | JCP 1101 | 600-0002 |
 | Richmond B | JCP 1201 | 600-0004 |
 | Richmond C | JCP 1102 | 600-0006 |
